### ISSUE TITLE
[Bug]: Unable to load model: 'extended_drawers:drawer_template'

### ISSUE THREAD

**Issue by Mg138** [MattiDragon/ExtendedDrawers#41]
### Description of the issue

Drawers have no texture/model.

### Reproduction steps

Not sure, but here's the pack just in case: https://codeberg.org/madeline/gaycraft

### Mod version

1.3.5

### Operating System

Manjaro Linux x86_64

### Minecraft version

1.19.2

### Other mods

https://codeberg.org/madeline/gaycraft

### Server

Not tested

### Client

Not tested

### Logs

https://mclo.gs/z6gm3BR

### Extra Context

Related warnings/errors:

```
[10:55:02] [Worker-Main-10/WARN]: Unable to load model: 'extended_drawers:drawer_template' referenced from: extended_drawers:single_drawer#facing=north: java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
[10:58:37] [Worker-Main-10/WARN]: Unable to resolve texture reference: #missingno in extended_drawers:block/single_drawer
[10:58:37] [Worker-Main-10/WARN]: Unable to resolve texture reference: #missingno in extended_drawers:item/quad_drawer
[10:58:37] [Worker-Main-10/WARN]: Unable to resolve texture reference: #missingno in extended_drawers:block/quad_drawer
[10:58:37] [Worker-Main-10/WARN]: Unable to resolve texture reference: #missingno in extended_drawers:block/double_drawer
[10:58:37] [Worker-Main-10/WARN]: Unable to resolve texture reference: #missingno in extended_drawers:item/double_drawer
[10:58:37] [Worker-Main-10/WARN]: Unable to resolve texture reference: #missingno in extended_drawers:item/single_drawer
```

**Comment by MattiDragon** [comment]
This issue has been fixed in thonkutil version 2.15.5, but you have 2.15.4 installed. Adding the updated thonkutil jar should fix the problem.

**Comment by Mg138** [comment]
Ah, it didn't say it supports 1.19.2, so it didn't update. We love versioning.
